core/math/quick_hull.h

```diff
@@ -70,11 +70,11 @@ class QuickHull {
 
 private:
 	struct FaceConnect {
-		List<Face>::Element *left, *right = nullptr;
+		List<Face>::Element *left = nullptr, *right = nullptr;
```

Comment:
I'd split this (and the next) each on their own line to avoid such mistakes in the future.